src/lib/Auth.ts

```diff
@@ -11,6 +10,9 @@ import chalk from 'chalk';
 import { AVAILABLE_ENVS } from './Env';
 import ThemeService from './api/services/theme.service';
 import { getLocalBaseUrl } from '../helper/serve.utils';
+import Debug from './Debug';
+
+const SERVER_TIMER = 60000 * 2;
```

Comment:
Change this to 5 min as per task description

Reply:
This is resolved